MoveImage2/Color.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Form
+{
+    class Color
+    {
+        private string foregroundColor;
+
+        public Color(string foregrcolor)
+        {
+            this.foregroundColor = foregrcolor;
+        }
+
+        public void GetConsoleColor()
```

Comment:
This method is doing 2 things it get the color then checks is it correct color or no. Decouple the logic of them and make reusable single methods